kernel/src/fs/utils/page_cache.rs

```diff
@@ -514,8 +512,12 @@ pub trait CachePageExt {
     }
 
     fn alloc_zero() -> Result<CachePage> {
-        let page = Self::alloc()?;
-        page.writer().fill(0);
+        let meta = CachePageMeta {
+            state: AtomicPageState::new(PageState::Uninit),
+        };
+        let page = FrameAllocOptions::new()
+            .zeroed(true)
```

Comment:
I found the combination of `PageState::Uninit` and `zeroed = true` strange. It seems that a new `CachePage` should be in one of the following two states:
1. `PageState::Uninit` and `zeroed = false`;
2. `PageState::UpToDate` and `zeroed = true`.

Reply:
Make sense and revised